### DISCUSSION TITLE
Can not see Workload hub on my fabric portal.

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/Microsoft-Fabric-workload-development-sample#117] [Q&A] [answered]
I am a fabric admin under my tenant, But I still do not see workload hub on my fabric portal. Is there anything I need to do like enable workload hub for the tenant or something to see workload hub?

**Comment by dhrumil83** [comment, marked as answer]
https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/discussions/32
